tools/run_tests/run_tests.py

```diff
@@ -191,19 +191,25 @@ def __init__(self):
       self._tests = json.load(f)
 
   def test_specs(self, config, travis):
-    modules = [config.job_spec(['tools/run_tests/run_python.sh', '-m',
```

Comment:
I don't believe that the tooling changes are particularly large and don't think we'd be gaining much from separating them out into more commits.